### PR TITLE
Loadout Departments Fix

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR fixes Zavodskoi not having engineering listed as a possible department in the loadout.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

File: code/game/jobs/faction/nanotrasen.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	Nonetheless, NanoTrasen has managed to secure itself as a crucial member of the newly-founded Stellar Corporate Conglomerate
 	allowing themselves to remain as a dominant corporate presence within the Orion Spur.
 	</p>"}
-	departments = {"Medical<br>Science<br>Service"}
+	departments = {"Service<br>Medical<br>Research"}
```
Comment (alsoandanswer) [inline code/game/jobs/faction/nanotrasen.dm:11]: Unnecessary change. Also, keep it as Science, as it already is in game and on the wiki. Don't make it confusing.

File: code/game/jobs/faction/pmc.dm [hunk 2]
```diff
@@ -9,7 +9,7 @@
 	Only some liaisons and bureaucrats work behind the scenes to hire and manage the contractors;
 	the rest of its members are part of several organizations contracted to supply the PMCG.
 	</p>"}
-	departments = {"Medical<br>Security"}
+	departments = {"Security<br>Medical"}
```
Comment (alsoandanswer) [inline code/game/jobs/faction/pmc.dm:12]: As above.

File: code/game/jobs/faction/zavodskoi.dm [hunk 3]
```diff
@@ -8,7 +8,7 @@
 	military force across the Orion Spur. They are the main corporation found in the Empire of
 	Dominia, and are at the forefront of weapons development technology.
 	</p>"}
-	departments = {"Science<br>Security"}
+	departments = {"Security<br>Engineering<br>Research"}
```
Comment (alsoandanswer) [inline code/game/jobs/faction/zavodskoi.dm:11]:
```suggestion
	departments = {"Science<br>Security<br>Engineering"}
```

File: code/game/jobs/faction/zeng_hu.dm [hunk 4]
```diff
@@ -11,7 +11,7 @@
 	controversial field that is now more accepted today.
 	</p>"}
 
-	departments = {"Medical<br>Science"}
+	departments = {"Medical<br>Research"}
```
Comment (alsoandanswer) [inline code/game/jobs/faction/zeng_hu.dm:14]: Unnecessary change.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]

**Review by JohnWildkins — APPROVED** [review]
yeah, technically

**Review by alsoandanswer — APPROVED** [review]

**Comment by alsoandanswer** [comment]
me face when the wiki page i was looking at was outdated
